waspc/src/Wasp/Analyzer/TypeDefinitions/TH/HasCustomEvaluation.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+
+module Wasp.Analyzer.TypeDefinitions.TH.HasCustomEvaluation
+  ( HasCustomEvaluation (..),
+  )
+where
+
+import Data.Typeable (Typeable)
+import Wasp.Analyzer.Evaluator.Evaluation.TypedExpr (TypedExprEvaluation)
+import Wasp.Analyzer.Type (Type)
+
+-- | TODO: Put this class in a better place? Maybe in Evaluator?
```

Comment:
Maybe I should move the whole TH back to Evaluator? Uff I need to rethink this again.